samples/android/camera-calibration/src/org/opencv/samples/cameracalibration/CameraCalibrationActivity.java

```diff
@@ -136,23 +136,22 @@ public boolean onPrepareOptionsMenu (Menu menu) {
 
     @Override
     public boolean onOptionsItemSelected(MenuItem item) {
-        switch (item.getItemId()) {
```

Comment:
Related to: https://stackoverflow.com/questions/26753328/switch-case-with-r-id-xxxx-in-android-doesnt-work